source/AI.cpp

```diff
@@ -2037,11 +2095,30 @@ bool AI::ShouldDock(const Ship &ship, const Ship &parent, const System *playerSy
 	if(!requiredAmmo.empty())
 		return true;
 
+	// Reboard if low power/no power (battery only).
+	if(ship.IsEnergyLow())
+		return true;
+
+	// Reboard/retreat if harmless and enemy nearby; for example boxwings.
```

Comment:
I know technically you don't need to do this, but that is content style...
```suggestion
	// Reboard/retreat if harmless and enemy nearby; for example Boxwings.
```